configure.cmake

```diff
@@ -161,6 +161,33 @@ IF(UNIX)
       SET(LIBWRAP "wrap")
     ENDIF()
   ENDIF()
+
+  SET(WITH_SYSTEMD "auto" CACHE STRING "Compile with systemd notification on ready")
+  IF(NOT ${WITH_SYSTEMD} MATCHES "[Nn][Oo]")
```

Comment:
Please do not accept unknown values as "auto". E.g. if one mistakenly does WITH_SYSTEM=off or WITH_SYSTEMD=0.